rest_framework/authtoken/management/commands/drf_create_token.py

```diff
@@ -16,5 +16,11 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         username = options['username']
-        token = self.create_user_token(username)
+
+        try:
+            token = self.create_user_token(username)
+        except User.DoesNotExist:
+            print('Cannot create the Token: user {0} does not exist'.format(
```

Comment:
Not entirely sure if it's relevant, but management commands typically use `self.stdout.write()` instead of `print()`. eg, https://github.com/django/django/blob/1.11/django/core/management/commands/migrate.py#L141